### PR TITLE
Create review model

### PR DESCRIPTION
**Pull request by bdoughertys**
Set up model with user service dependency. Crash tested, deleting user_service deletes review.

### CODE REVIEW

File: app/models/review.rb [hunk 1]
```diff
@@ -1,2 +1,8 @@
 class Review < ApplicationRecord
+  belongs_to :user_service, dependent: :destroy
+  belongs_to :user
```
Comment (williamgrant04) [inline app/models/review.rb:3]: Try `belongs_to :user, through: :user_service` to allow for `review.user` instead of `review.user_service.user`

**Review by williamgrant04 — CHANGES_REQUESTED** [review]

File: app/models/review.rb (hunk 1 above)
Comment (bdoughertys) [reply app/models/review.rb:3]: This would make the review belong to the user of user_service rather than the user leaving the review, no?

Comment (williamgrant04) [reply app/models/review.rb:3]: Yeah good point

**Review by williamgrant04 — APPROVED** [review]
👍

**Pull request merged**